src/components/Layout/FormPageLayout.tsx

```diff
@@ -56,27 +58,25 @@ function FormPageLayoutComponent<F>(props: FormPageLayoutProps<F>) {
     [formSections],
   );
 
-  // The grid columns are defined as: "left-gutter, left-nav, form-content, right-sidebar, right-gutter"
-  const gridColumns =
-    "minMax(0, auto) minMax(100px, 250px) minMax(350px, 1000px) minMax(min-content, 300px) minMax(0, auto)";
+  // The grid columns are defined as: "left-nav, form-content, right-sidebar"
+  const gridColumns = `minMax(100px, 250px) minMax(350px, 1000px) minMax(${RIGHT_SIDEBAR_MIN_WIDTH}, 380px)`;
 
   return (
     // This page is `fixed` to the full screen to allow it to act as a full screen modal while content is mounted below
     // since this layout will be replacing most superdrawers/sidebars, we keep the listing mounted below to preserve the users's
     // scroll position & filters
     // Adding "align-items: start" allows "position: sticky" to work within a grid for the sidebars
-    <div
```

Comment:
The material change here is splitting the page container into two:
* Outer `fixed` container that behaves like a full screen modal
  *  Display flex with justify content center so the inner grid is centered un ultra-wide screens
* Inner `grid` container that is bound by the `maxContentWidthPx`